### DISCUSSION TITLE
LTSP with the smallest image size, let say <1GB

### DISCUSSION THREAD

**Discussion by awanbandung** [ltsp/ltsp#725] [Q&A] [answered]
hi there,

i have multiple client, around >100 user, they only use chrome for our applications..
so actually the OS for load chrome in Kiosk Mode. and some nvdia drivers.

client want load faster, the client already has NIC 10Gbps, CPU intel i7-6th and GPU nvidia 1060 ti
which the best OS for that requirements, i try Lubuntu but get 2GB ++ image size.

is that possible has image less than 2GB? which OS? 

pls advice.

**Comment by alkisg** [comment]
Hi, the image size doesn't affect the LTSP client boot times or the application load times. You may have an 100 GB LTSP image and the LTSP clients can boot from it and load the kiosk in e.g. 20 seconds.

Or do you mean that you're using IMAGE_TO_RAM and you're loading all the image to RAM?

In any case, you can just create a chroot with `debootstrap` and you can install ltsp, chrome and the nvidia drivers in it; apt will automatically take care of the dependencies and give you the minimal possible image; I suspect it will be a lot less than 2 GB.

**Reply by awanbandung** [reply]
i run with chrootless, and use PIB-ITR from [this](https://github.com/ltsp/ltsp/wiki/Client-Choices-and-Performance) instead IMAGE_TO_RAM, if the image 100 GB , the client  will download the image 100 GB at ipxe 1st load. ?

**Comment by alkisg** [comment, marked as answer]
If you use ...PIB-ITR or ITR, then yes you download the whole 100 GB image so it's very slow. It's not downloaded at ipxe but later on.

While if you use the LTSP defaults, you only download 150 MB (Megabyte) for the client to boot, even if the image is 100 GB, so it's very fast.

Btw that wiki page says that the new LTSP is slower than LTSP5 at the first launch of each app. This isn't true as the same technologies are used in both LTSP versions (nfs/squashfs/sshfs).
